static/js/reducers/dashboard.js

```diff
@@ -36,7 +36,11 @@ export const dashboard = (state: DashboardsState = INITIAL_DASHBOARDS_STATE, act
   case REQUEST_DASHBOARD: // eslint-disable-line no-case-declarations
     let newBaseState = R.dissoc(username, state);
     if (action.payload === true) {
-      return updateDashboardState(newBaseState, username, INITIAL_DASHBOARD_STATE);
+      return updateDashboardState(
+        newBaseState,
+        username,
+        _.merge({}, INITIAL_DASHBOARD_STATE, { fetchStatus: null })
```

Comment:
what's the reason for this change? the `newBaseState` shouldn't have anything saved in it under the `username` and `INITIAL_DASHBOARD_STATE` doesn't have a `fetchStatus` key on it.